jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/ProcessInstanceLogPresenter.java

```diff
@@ -167,17 +191,70 @@ public void loadTaskDetails(Long workItemId,
     public void onProcessInstanceSelectionEvent(@Observes final ProcessInstanceSelectionEvent event) {
         this.serverTemplateId = event.getServerTemplateId();
         this.containerId = event.getDeploymentId();
-        resetLogsList();
-        dataSetQueryHelper.setCurrentTableSettings(filterSettingsManager.createDefaultFilterSettingsPrototype(event.getProcessInstanceId()));
+        if (logsDataSetQueryHelper.getCurrentTableSettings() == null) {
+            logsDataSetQueryHelper.setCurrentTableSettings(
+                    filterSettingsManager.createDefaultFilterSettingsPrototype(event.getProcessInstanceId()));
+            setupDefaultActiveSearchFilters();
+        } else {
+            refreshGrid();
+        }
+    }
+
+    public void refreshGrid() {
+        currentPage = 0;
         loadProcessInstanceLogs();
     }
 
+    @Override
+    public void setupDefaultActiveSearchFilters() {
+        processInstanceLogBasicFiltersPresenter.onClearAllActiveFiltersEvent(new ClearAllActiveFiltersEvent());
+        final List<String> types = Arrays.asList(LogUtils.NODE_TYPE_START,
+                                                 LogUtils.NODE_TYPE_END,
+                                                 LogUtils.NODE_TYPE_HUMAN_TASK);
+        final List<String> nodeTypeLabels = Arrays.asList(constants.StartNode(),
+                                                          constants.EndNode(),
+                                                          constants.Human_Tasks());
+        addActiveFilter(in(COLUMN_LOG_NODE_TYPE,
+                           types),
+                        constants.EventNodeType(),
+                        String.join(", ",
+                                    nodeTypeLabels),
+                        types,
+                        v -> removeActiveFilter(in(COLUMN_LOG_NODE_TYPE,
+                                                   types))
+        );
+
+        addActiveFilter(in(COLUMN_LOG_TYPE,
+                           Collections.emptyList()),
+                        constants.EventType(),
+                        "",
+                        Collections.emptyList(),
+                        v -> removeActiveFilter(in(COLUMN_LOG_TYPE,
+                                                   Collections.emptyList())));
+    }
+
+    @Override
+    public void createListBreadcrumb() {
```

Comment:
It is a bit of boilerplate code in here and view because of the new superclass, but not exactly sure what would be the best way to avoid this.